ui/components/app/modals/index.scss

```
@@ -35,7 +35,7 @@
     bottom: 0;
     left: 0;
     z-index: 1040;
-    background-color: #373a47;
+    background-color: #373a47; /* TODO: What color here? */
```

@georgewrmarshall @GuillaumeRx What color would be best here?

---

As it's a background color I would suggest `var(--color-overlay-alterantive)` `--color-text-default` shouldn't be used for any background-color really. I'm trying to find out how and where this is used.

---

Doesn't look like this is even being used but I think it should be and overlay. I would suggest updating it to
```suggestion
    background-color: var(--color-overlay-alterantive);
```